src/test/resources/unit/fix-test/expected/src/main/java/fix/test/ClassWithJavadoc.java

```diff
@@ -44,6 +45,25 @@ public ClassWithJavadoc( String aString )
     {
     }
 
+    /**
+     * public constructor with annotation
+     *
+     * @param b a {@link java.lang.Boolean} object.
```

Comment:
You don't need these tags in test resources, though the above comment is helpful

Reply:
@elharo
Yes, these functions are modified from my lib's functions where I meet this bug, I just forgot to delete the `@see` tags.
deleted now.